recipe/meta.yaml

```diff
@@ -38,11 +38,11 @@ test:
     - ipywidgets
   commands:
     - pip check
-    - coverage run --source ipywidgets --branch -m pytest -vv --color=yes --tb=long --pyargs ipywidgets 
+    - coverage run --source ipywidgets --branch -m pytest -vv --color=yes --tb=long --pyargs ipywidgets
     - coverage report --show-missing --skip-covered --fail-under=90
 
 about:
-  home: https://github.com/ipython/ipywidgets
```

Comment:
ha, looks like upstream even uses this in a few places:

https://github.com/jupyter-widgets/ipywidgets/blob/b78de43e12ff26e4aa16e6e4c6844a7c82a8ee1c/README.md?plain=1#L7